src/com/ppm/integration/agilesdk/connector/octane/OctaneRequestIntegration.java

```diff
@@ -133,11 +131,8 @@ private List<AgileEntityFieldInfo> transferModel(List<FieldInfo> fields) {
     }
 
     private List<FieldInfo> getWildcardFields(final String entityType, ClientPublicAPI client) {
-        List<SharedSpace> sharedSpacesList = client.getSharedSpaces();
-        if (!sharedSpacesList.isEmpty()) {
-            // always get the first shared space as one API token can only
-            // access to one space.
-            SharedSpace space = sharedSpacesList.get(0);
+        SharedSpace space = client.getActiveSharedSpace();
+        if (null != space) {
```

Comment:
We usually put it in the other order for ease of code reading: if (space != null)